src/PlexRequests.Store/BaseRepository.cs

```diff
@@ -0,0 +1,20 @@
+﻿using MongoDB.Bson;
+using MongoDB.Driver;
+
+namespace PlexRequests.Store
+{
+    public class BaseRepository<T>
```

Comment:
Doesn't this need to be abstract or something?

Reply:
Good point. At the moment, no, in future, it may well be. I'll leave as is for now.